server/handler/base.go

```diff
@@ -151,17 +163,11 @@ func (b *Base) Evaluate(ctx context.Context, installationID int64, trigger commo
 		return err
 	}
 
-	fetchedConfig, err := b.ConfigFetcher.ConfigForPR(ctx, prctx, client)
-	if err != nil {
-		return errors.WithMessage(err, fmt.Sprintf("Failed to fetch configuration: %s", fetchedConfig))
-	}
-
+	fetchedConfig := b.ConfigFetcher.ConfigForPR(ctx, prctx, client)
```

Comment:
just checking: we don't return an error anymore here because if one exists, its embedded in the `fetchedConfig` and is then evaluated below in `ValidateFetchedConfig` with better errors?

Reply:
Yes, previously, loading errors were handled by failing and not posting anything on the pull request. Only parsing errors resulted in a status check. In the new model, both types of errors are handled the same way.